### DISCUSSION TITLE
Workspace Ordering

### DISCUSSION THREAD

**Discussion by rin-senpai** [nikitabobko/AeroSpace#884] [questions-and-answers] [answered]
I believe workspaces are just ordered alphabetically, but is it possible to change that? For example, I want to have workspace 0 after 9 (since it's after it on the keyboard).

**Comment by lucidsleeping** [comment]
Similar issue, 

Steps to reproduce:
1.Edit config to use Q,W,E,R

```
alt-q = 'workspace Q'
alt-w = 'workspace W'
alt-e = 'workspace E'
alt-r = 'workspace R'

```
Expected result: Ordering to stay in order of assignment in config
Actual result: Alphabetical ordering

<img width="191" height="47" alt="Screenshot 2025-12-19 at 14 41 33" src="https://github.com/user-attachments/assets/64ec6572-75bd-4833-a2bf-d1b29b112914" />

**Comment by nikitabobko** [comment, marked as answer]
https://github.com/nikitabobko/AeroSpace/issues/1816
